dockers/tpu-tests/tpu_test_cases.jsonnet

```diff
@@ -10,17 +10,31 @@ local tputests = base.BaseTest {
 
   timeout: 900, # 15 minutes, in seconds.
 
-  image: std.extVar('image'),
-  imageTag: std.extVar('image-tag'),
+  image: 'pytorchlightning/pytorch_lightning:base-xla-py3.6-torch{PYTORCH_VERSION}',
+  imageTag: '{PR_NUMBER}-{PYTORCH_VERSION}',
 
   tpuSettings+: {
-    softwareVersion: 'pytorch-VERSION',
+    softwareVersion: 'pytorch-{PYTORCH_VERSION}',
   },
   accelerator: tpus.v3_8,
 
   command: utils.scriptCommand(
     |||
+      mkdir -p /home/runner/work/pytorch-lightning && cd /home/runner/work/pytorch-lightning
+      git clone https://github.com/PyTorchLightning/pytorch-lightning.git
       cd pytorch-lightning
+      echo $PWD
+      git ls-remote --refs origin
+      git fetch origin "refs/pull/{PR_NUMBER}/head:pr/{PR_NUMBER}" && git checkout "pr/{PR_NUMBER}"
```

Comment:
Any reasons why this would be needed, as the file is edited in the ci directly.

<img width="675" alt="Screenshot 2021-02-22 at 09 55 36" src="https://user-images.githubusercontent.com/12861981/108692104-1ea67600-74f4-11eb-82ed-4a5711cdda0a.png">